app/routes/forum/categories/category/threads/thread/posts/index.js

```diff
@@ -68,6 +68,13 @@ export default class PostIndexRoute extends AuthenticatedRoute {
     this.controller.send('goToNextPage');
   }
 
+  resetController(controller, isExiting, transition) {
```

Comment:
https://api.emberjs.com/ember/4.8/classes/Route/methods/resetController?anchor=resetController for docs on this hook